### ISSUE TITLE
[Feature] Studio ownership history and studio code format

### ISSUE THREAD

**Issue by sleetx** [stashapp/stash-box#820]
A common issue with tracking Studio content is understanding the studio's ownership. Over time, studios may operate under different ownership or distribution groups, or the studio goes defunct and their library gets sold to 3rd parties.

I suggest adding a section to the Studio page where we can enter **what years the studio was actively releasing content** as well as **date ranges of their ownership**. This will assist in identifying the *original* source of studio content compared to re-releases. It will also help answer questions like "where can I find this defunct studio's content today?"

In addition, let's add a place to input the studio's **scene code format**, which can assist with scraping and identification.

**Comment by echo6ix** [comment]
> this will assist in identifying the original source of studio content compared to re-releases.

Wouldn't this be creating redundancy since that goal is precisely what #663 -- Release Groups aka Master Release & Versions -- aims to accomplish, using a paradigm that Discogs and MusicBrainz have more-or-less perfected. 

Every object has a master release (the original source), and all derivative releases are associated back to the master, creating a historical chain of versions that always flow back to the original source.

Or are you requesting a type of Release Groups paradigm specifically for studios as a way to track their genealogy so to speak? That would be interesting, and probably useful, since I think anytime a studio/site/company changes their name or brand, they should technically get a new corroborating entry to match the change rather than retroactively changing the name associated with past releases. I believe from an archival standpoint that would be the purist/correct way.

**Comment by sleetx** [comment]
> Or are you requesting a type of Release Groups paradigm specifically for studios as a way to track their genealogy so to speak? That would be interesting, and probably useful, since I think anytime a studio/site/company changes their name or brand, they should technically get a new corroborating entry to match the change rather than retroactively changing the name associated with past releases. I believe from an archival standpoint that would be the purist/correct way.

Yes this is more about studios as a whole, and being able to identify their status in any given year. I'm thinking of something like a genealogical timeline... It would help give a complete picture of when, where, and why their content is being published or re-published.

Let's take a generic "Studio X" for example, and how StashBox could track the life and ownership of that studio. Here's an example:


**Studio X ~ Active: 2000-2012**

Ownership timeline:
- 2000-2005: **Studio X**... _(self-managed)_
- 2005-2010: **Studio XYZ Group**... _(rolled into a larger entity that manages related studios)_
- 2010-2020: **Big Media Network**... _(sold to this entity, which shut down the original studio in 2012. this company maybe re-publishes some old content under a different name)_
- 2020 - Present: **Archival Media Site**... _(sold to this entity. although "Studio X" no longer exists, their back catalog is available here)_

**Comment by sleetx** [comment]
If this is too big of an ask, a simple multi-line "Notes" field on the studio page would probably work.

But if there is interest in a larger Studio page enhancement, I'd recommend the ownership history fields alongside the ones requested in #676

**Comment by echo6ix** [comment]
> If this is too big of an ask, a simple multi-line "Notes" field on the studio page would probably work.

A free form notes field will quickly devolve into inconsistent and unstructured input with a community driven project like this, where you have hundreds of chefs in the kitchen, most of which won't be familiar with guidelines/rules/established protocol.

Your name history suggestion that advocates for a structured schema with clearly defined fields are much, much better.

**Comment by AdultSun** [comment]
> A free form notes field will quickly devolve into inconsistent and unstructured input with a community driven project like this, where you have hundreds of chefs in the kitchen, most of which won't be familiar with guidelines/rules/established protocol.

Wouldn't be as much of an issue if the notes field had limited permissions. For example, if only mods/admins could make or approve any changes. But I realize that's also a different feature request than this particular thread.
